### PR TITLE
Add Aqua check

### PR DESCRIPTION
**Pull request by hkaj**
### What does this PR do?

Create an Aqua check.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: aqua/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Agent Check: Aqua
+
+## Overview
+
+This check monitors [Aqua][1].
+
```
Comment (dsahni) [inline aqua/README.md]:
```suggestion

The Aqua check will alert the user if total high-severity vulnerability is reached, or if a container is running inside a host not registered by Aqua.  Aqua will also send data alerts regarding blocked events in runtime, and it is possible to trigger a webhook to scale infrastructure if more Aqua scanners are required.

```

File: aqua/manifest.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "display_name": "Aqua",
+  "maintainer": "oran.moshai@aquasec.com",
+  "manifest_version": "1.0.0",
+  "name": "aqua",
+  "metric_prefix": "aqua.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Datadog support for Aqua monitoring",
```
Comment (dsahni) [inline aqua/manifest.json]:
```suggestion
  "short_description": "Full dev-to-prod security solution for containers and cloud native applications",
```

File: aqua/metadata.csv [hunk 3]
```diff
@@ -0,0 +1,7 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+aqua.images,gauge,,image,,The number of images seen by Aqua,0,aqua,images,
```
Comment (dsahni) [inline aqua/metadata.csv]:
```suggestion
aqua.images,gauge,, unit,,The number of images seen by Aqua,0,aqua,images,
```

Comment (dsahni) [reply aqua/metadata.csv]: `image` isn't currently a supported unit type

File: aqua/metadata.csv [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+aqua.vulnerabilities,gauge,,vulnerability,,The number and categories of vulnerabilities detected by Aqua,-1,aqua,vulns,
```
Comment (dsahni) [inline aqua/metadata.csv]:
```suggestion
aqua.vulnerabilities,gauge,, occurrence,,The number and categories of vulnerabilities detected by Aqua,-1,aqua,vulns,
```

Comment (dsahni) [reply aqua/metadata.csv]: `vulnerability` isn't currently a supported unit type

File: aqua/metadata.csv [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+aqua.running_containers,gauge,,container,,The number of running containers seen by Aqua,0,aqua,running containers,
+aqua.audit.access,gauge,,access,,The number of audit events per category,0,aqua,audit access,
```
Comment (dsahni) [inline aqua/metadata.csv]:
```suggestion
aqua.audit.access,gauge,,event,,The number of audit events per category,0,aqua,audit access,
```

File: aqua/metadata.csv [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+aqua.scan_queue,gauge,,queue,,The number of scan queues per type,0,aqua,scan Q,
```
Comment (dsahni) [inline aqua/metadata.csv]:
```suggestion
aqua.scan_queue,gauge,,occurrence,,The number of scan queues per type,0,aqua,scan Q,
```

File: aqua/metadata.csv [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+aqua.enforcers,gauge,,enforcer,,The number of host enforcers per status,0,aqua,enforcers,
```
Comment (dsahni) [inline aqua/metadata.csv]:
```suggestion
aqua.enforcers,gauge,,host,,The number of host enforcers per status,0,aqua,enforcers,
```

File: aqua/metadata.csv [hunk 8]
```diff
@@ -0,0 +1,7 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+aqua.images,gauge,, unit,,The number of images seen by Aqua,0,aqua,images,
+aqua.vulnerabilities,gauge,, occurrence,,The number and categories of vulnerabilities detected by Aqua,-1,aqua,vulns,
+aqua.running_containers,gauge,,container,,The number of running containers seen by Aqua,0,aqua,running containers,
+aqua.audit.access,gauge,,event,,The number of audit events per category,0,aqua,audit access,
+aqua.scan_queue,gauge,,occurrence,,The number of scan queues per type,0,aqua,scan Q,
+aqua.enforcers,gauge,,host,,The number of host enforcers per status,0,aqua,enforcers,
```
Comment (mfpierre) [inline aqua/metadata.csv]: There are trailing commas on all the metric lines that should be removed

File: aqua/metadata.csv [hunk 9]
```diff
@@ -0,0 +1,7 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+aqua.images,gauge,, unit,,The number of images seen by Aqua,0,aqua,images,
```
Comment (mfpierre) [inline aqua/metadata.csv]:
```suggestion
aqua.images,gauge,,unit,,The number of images seen by Aqua,0,aqua,images
```

File: aqua/metadata.csv [hunk 10]
```diff
@@ -0,0 +1,7 @@
...
+aqua.vulnerabilities,gauge,, occurrence,,The number and categories of vulnerabilities detected by Aqua,-1,aqua,vulns,
```
Comment (mfpierre) [inline aqua/metadata.csv]:
```suggestion
aqua.vulnerabilities,gauge,,occurrence,,The number and categories of vulnerabilities detected by Aqua,-1,aqua,vulns
```

File: aqua/tests/test_aqua.py [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+import os
+import json
+
+import pytest
+from mock import MagicMock
+
+from datadog_checks.aqua import AquaCheck
+
+
+CHECK_NAME = 'sortdb'
```
Comment (mfpierre) [inline aqua/tests/test_aqua.py]:
```suggestion
CHECK_NAME = 'aqua'
```

Comment (mfpierre) [reply aqua/tests/test_aqua.py]: but I guess this is unused, so probably could be removed

**Pull request merged**